pkg/unittest/input/case-1-awsconfig.golden

```diff
@@ -3,5 +3,6 @@ kind: AWSConfig
 metadata:
   labels:
     "giantswarm.io/service-priority": "highest"
+    giantswarm.io/organization: my-organization
```

Comment:
New CAPI agent code needs an organization

Reply:
Shouldn't this line be added to the case-5-cluster-api-v1alpha3.golden as well ?

Reply:
Ok ! Done